engine/message/db.go

```diff
@@ -277,6 +279,23 @@ func NewDB(ctx context.Context, db *sql.DB, c *Config, a alertlog.Store) (*DB, e
 			) select distinct msg_id, alert_id, user_id, cm_id from disabled
 		`),
 
+		twilioDisabled: p.P(`
+			with disabled as (
+				update outgoing_messages msg
+				set
+					last_status = 'failed',
+					last_status_at = now(),
+					status_details = 'twilio disabled',
+					cycle_id = null,
+					next_retry_at = null
+				from user_contact_methods cm
+				where
+					msg.last_status = 'pending' and
+					msg.message_type != 'verification_message'
+				returning msg.id as msg_id, alert_id, msg.user_id, cm.id as cm_id
+			) select distinct msg_id, alert_id, user_id, cm_id from disabled
```

Comment:
I think the only reason this was needed is we weren't joining properly, it should be enough to just use the `update...returning`